week-1/H-numbers/exercise.js

```diff
@@ -1 +1,9 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var num = "Number of ";
+var sum = numberOfStudents + numberOfMentors;
+var message = "Total number of students and mentors: ";
+console.log(num + "students: " + numberOfStudents);
+console.log(num + "mentors: " + numberOfMentors);
+console.log(message + sum);
```

Comment:
Having a variable called `num` which contains the string "Number of " seems a bit complicated. I'd expect it to be a number.  It's probably simpler just to write "Number of students" and "Number of mentors" as separate constant strings here. Don't worry that they contain repeated text.